crates/router/src/connector/stripe.rs

```diff
@@ -1784,12 +1845,18 @@ impl
         req: &types::SubmitEvidenceRouterData,
         _connectors: &settings::Connectors,
     ) -> CustomResult<Vec<(String, request::Maskable<String>)>, errors::ConnectorError> {
-        let mut header = vec![(
-            headers::CONTENT_TYPE.to_string(),
-            types::SubmitEvidenceType::get_content_type(self)
-                .to_string()
-                .into(),
-        )];
+        let mut header = vec![
```

Comment:
@swangi-kumari Let's keep this logic inside get_auth_header method which is being used in all get_headers.